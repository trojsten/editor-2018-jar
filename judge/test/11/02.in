STR:
General
RAWKAMZ
STR:
Plukovnik
XQUWCX
STR:
Podplukovnik
DAPSX
STR:
Strateg
NZBIUMZ
STR:
Spion
JTKGZMQ
STR:
Programator
T
STR:
Velmajstermaxisupergeneralveduci
MOC
STR:
Nadpodpredzadplukovnikazarovendokoncaajgeneral
XXMI
