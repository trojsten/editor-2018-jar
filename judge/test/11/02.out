{"Velmajstermaxisupergeneralveduci": "XXMI", "Spion": "T", "Nadpodpredzadplukovnikazarovendokoncaajgeneral": "MOC", "Programator": "XQUWCX", "General": "NZBIUMZ", "Strateg": "JTKGZMQ", "Podplukovnik": "RAWKAMZ", "Plukovnik": "DAPSX"}